int32 id
float64 confidence
geometry_msgs/Quaternion quaternion
float64[] u_corners
float64[] v_corners
